flink-clients/src/test/java/org/apache/flink/client/CliFrontendAddressConfigurationTest.java

```diff
@@ -45,163 +43,122 @@
 
 	@Rule
 	public TemporaryFolder folder = new TemporaryFolder();
-	
+
 	@BeforeClass
 	public static void init() {
 		CliFrontendTestUtils.pipeSystemOutToNull();
 	}
-	
+
 	@Before
-	public void clearConfig() {
+	public void clearConfig() throws Exception {
 		CliFrontendTestUtils.clearGlobalConfiguration();
 	}
 
 	@Test
-	public void testInvalidConfigAndNoOption() {
+	public void testInvalidConfigAndNoOption() throws Exception {
+		CliFrontend frontend = new CliFrontend(CliFrontendTestUtils.getInvalidConfigDir());
+		CommandLineOptions options = mock(CommandLineOptions.class);
+
 		try {
-			CliFrontend frontend = new CliFrontend(CliFrontendTestUtils.getInvalidConfigDir());
-			CommandLineOptions options = mock(CommandLineOptions.class);
-
-			try {
-				frontend.getJobManagerAddress(options);
-				fail("we expect an exception here because the we have no config");
-			}
-			catch (Exception e) {
-				// expected
-			}
+			frontend.getJobManagerAddress(options);
+			fail("we expect an exception here because the we have no config");
 		}
 		catch (Exception e) {
-			e.printStackTrace();
-			fail(e.getMessage());
+			// expected
```

Comment:
Ah ok, this makes sense. Then I guess it's a problem of too unspecific exceptions which we throw in our code. But this seems to be a general issue.